### PR TITLE
Copyedit to service:push content

### PR DESCRIPTION
**Pull request by StephenBarlow**

### CODE REVIEW

File: graph-manager-docs/source/federation.mdx [hunk 1]
```diff
@@ -123,38 +123,34 @@ server.listen().then(({ url }) => {
 
 > The managed configuration will default to using the `'current'` variant.  When running Apollo Gateway in an environment where outbound traffic to the internet is restricted, consult the [directions for configuring a proxy](https://www.apollographql.com/docs/apollo-server/proxy-configuration/) within Apollo Server.
 
-### Managed configuration
+## Managed configuration
 
-Managed configuration is the concept of allowing the gateway to update dynamically in response to service changes. In principle, it's important to separate the reliability of your graph from the reliability of your services. For that reason, we recommend running federation in production using a managed configuration, where the gateway picks up config changes not from introspection, but from a set of files owned by your graph describing its current state.
+With **managed configuration**, your GraphQL gateway updates dynamically in response to schema changes in your implementing services. You make these service changes with the `apollo service:push` CLI command, which pushes your service configuration files to Graph Manager and stores them securely.
```
Comment (zionts) [inline graph-manager-docs/source/federation.mdx]: Should we reiterate that you need to pass `--serviceName` to the `apollo service:push` command?

File: graph-manager-docs/source/federation.mdx [hunk 2]
```diff
@@ -123,38 +123,34 @@ server.listen().then(({ url }) => {
...
-queries that fail validation in downstream services or violate resolvers'
-expectations. Some examples of this include:
+### Pushing configuration updates safely
+
+Whenever possible, you should update your service configuration in a way that is backward compatible to avoid downtime. To help ensure this, **call `apollo service:push` for an implementing service only after all replicas of the service are deployed**. This ensures that resolvers are in place for all operations that are executable against your graph, and operations can't access fields that do not yet exist.
+
+In the rare case where a configuration change is not backward compatible with your gateway's query planner, you should update your service registry _before_ you deploy your implementing services.
+
+Additionally, you should perform configuration updates that affect query planning separately from (and prior to) other changes. This helps avoid a scenario where the query planner generates queries that fail validation in downstream services or violate your resolvers.
```
Comment (zionts) [inline graph-manager-docs/source/federation.mdx]: Perhaps something like "In short, when making changes that may affect the query planner, tread carefully, and consider how changes will propagate across services"?

File: graph-manager-docs/source/federation.mdx [hunk 3]
```diff
@@ -123,38 +123,34 @@ server.listen().then(({ url }) => {
...
+
+Examples of this include:
 
 1. Modifying `@key` or `@requires` dependencies while simultaneously modifying dependency fields
```
Comment (zionts) [inline graph-manager-docs/source/federation.mdx]: We could just make this "Modifying `@key`, `@requires` or `@provides` directives"

File: graph-manager-docs/source/federation.mdx [hunk 4]
```diff
@@ -222,12 +222,11 @@ Currently, the Apollo gateway expands all interfaces into implementing types,
 though a follow-up to the query-planning logic may make this only occur for
 interfaces that serve as entities (i.e. have a `@key`).
 
-#### Modifying query-planning logic
+### Modifying query-planning logic
+
+Treat migrations of your query-planning logic similarly to how you treat database migrations. Carefully consider the effects on downstream services as the query planner changes, and plan for "double reading" as appropriate.
```
Comment (zionts) [inline graph-manager-docs/source/federation.mdx]: Oh hey, this is the sentence I wanted. Yay!

File: graph-manager-docs/source/federation.mdx (hunk 1 above)
Comment (StephenBarlow) [reply graph-manager-docs/source/federation.mdx]: Including a codeblock here that includes the flag. Really what we need is a definitive command reference to link to in all of these cases, which is a short-to-mid-term goal

Comment (StephenBarlow) [reply graph-manager-docs/source/federation.mdx]: ok actually linking elsewhere in the article where an applicable codeblock is already shown

**Pull request merged**